### DISCUSSION TITLE
What about Deno?

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#91] [Q&A]
Deno doesn't support [preloading scripts](https://github.com/denoland/deno/issues/713). This makes Deno a non-starter since the entire idea of Script Kit's api is based on [preloading](https://github.com/johnlindquist/kit/blob/main/script#L19).

I will definitely keep an eye out as Deno matures.

**Comment by alkene5** [comment]
> Deno doesn't support [preloading scripts](https://github.com/denoland/deno/issues/713). This makes Deno a non-starter since the entire idea of Script Kit's api is based on [preloading](https://github.com/johnlindquist/kit/blob/main/script#L19).
> 
> I will definitely keep an eye out as Deno matures.

https://deno.com/blog/v2.4#modify-the-deno-environment-with-the-new---preload-flag

Deno is definitely worth it—it’s better than Node anyway.

**Reply by alkene5** [reply]
Deno and Bun are both awesome—Deno has no node_modules and a fully modern toolchain, while Bun runs on an optimized JSC runtime and can outperform Node in many cases.
